docs/GettingStarted.md

```diff
@@ -1,7 +1,9 @@
 # Getting started with Calico on Docker
 
+>*Note that Calico uses Docker's [libnetwork network driver](https://github.com/docker/libnetwork), available in the Docker [experimental channel](https://github.com/docker/docker/tree/master/experimental) alongside the Docker 1.7 release.  This replaces Calico's use of powerstrip as a network plugin.   However, Docker's experimental channel is still moving fast and some of its features are not yet fully stable, so the stable powerstrip cluster configuration with CoreOS is still available [here](https://github.com/Metaswitch/calico-coreos-vagrant-example).*
```

Comment:
I'm not sure this should link to the vagrant example, but perhaps just to the getting started guide of the powerstrip branch.  wdyt?